src/runtime_src/core/tools/xbmgmt2/flash/xspi.cpp

```diff
@@ -1854,6 +1854,26 @@ static int writeToFlash(std::FILE *flashDev, int slave,
     return ret;
 }
 
+static int 
```

Comment:
```suggestion
void
```
The code should have been updated to throw and exception instead of using a return value.   Do you have time to clean this up?  If not, then could you create CR, assign it to yourself for 2022.2, with a high priority value.